uv.lock

```diff
@@ -1326,7 +1326,7 @@ wheels = [
 
 [[package]]
 name = "vortex-array"
-version = "0.22.1"
+version = "0.21.1"
```

Comment:
I don't understand why uv insists on changing this

Reply:
well, 0.20.0 was the last successful build, eh

Reply:
ugh yea, not sure. can revert

Reply:
I also got it locally so not sure if this lock is correct anymore

Reply:
I think we can also release, the python build should be fixed.